### PR TITLE
charts/wire-server: add setFederationDomain

### PR DESCRIPTION
**Pull request by arianvp**
This is a non-optional value in the next wire-server release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# NEXT
+
+A new mandatory option `optSettings.setFederationDomain` has been introduced to
+`brig` which in the future will be used for Wire federation.  This domain name
+is *not* optional even if federation is not used.
+
+Please update your `values/wire-server/values.yaml` to set `brig.optSettings.setFederationDomain`
```
Comment (jschaul) [inline CHANGELOG.md:7]:
```suggestion
Please update your `values/wire-server/values.yaml` to set `brig.optSettings.setFederationDomain` and `galley.optSettings.federationDomain` (note the slightly different option name).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# NEXT
+
+A new mandatory option `optSettings.setFederationDomain` has been introduced to
+`brig` which in the future will be used for Wire federation.  This domain name
```
Comment (jschaul) [inline CHANGELOG.md:4]:
```suggestion
`brig` and `galley` which in the future will be used for Wire federation.  This domain name
```

File: values/wire-server/demo-values.example.yaml [hunk 3]
```diff
@@ -40,6 +40,7 @@ brig:
       teamCreatorWelcome: https://teams.example.com/login # change this
       teamMemberWelcome: https://wire.example.com/download # change this
     optSettings:
+      setFederationDomain: example.com # change this
```
Comment (jschaul) [inline values/wire-server/demo-values.example.yaml:43]: also needs to be added to galley options.

File: values/wire-server/prod-values.example.yaml [hunk 4]
```diff
@@ -41,6 +41,8 @@ brig:
       teamSettings: https://teams.example.com # change this (or unset if team settings are not used)
       teamCreatorWelcome: https://teams.example.com/login # change this
       teamMemberWelcome: https://wire.example.com/download # change this
+    optSettings:
+      setFederationDomain: example.com # change this
```
Comment (jschaul) [inline values/wire-server/prod-values.example.yaml:45]: also galley

**Review by jschaul — COMMENTED** [review]
Thanks for making this PR: the change also needs to be added to galley, unfortunately.

File: values/wire-server/values.yaml [hunk 5]
```diff
@@ -0,0 +1,300 @@
+tags:
```
Comment (jschaul) [inline values/wire-server/values.yaml:1]: did you mean to commit this file?

File: values/nginx-ingress-services/values.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+accountPages:
+  enabled: true
+certManager:
+  certmasterEmail: arian.vanputten@gmail.com
```
Comment (jschaul) [inline values/nginx-ingress-services/values.yaml:4]: did you mean to commit this file?

**Review by jschaul — APPROVED** [review]
Two files accidentally committed here, please remove those; otherwise looks good!

**Review by jschaul — APPROVED** [review]

File: values/nginx-ingress-services/values.yaml (hunk 6 above)
Comment (arianvp) [reply values/nginx-ingress-services/values.yaml]: Nope I did not! thanks :)

**Pull request merged**